src/compiler/build.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 #MAKE=gmake -j4
-MAKE=make
+MAKE="make -j 8"
```

Comment:
Good idea, but not portable, since nproc seems to be a linux-specific command, neither FreeBSD, OSX nor M$ apparently to support it :(